data/other/ldap_scheme.yml

```diff
@@ -28,6 +28,12 @@ parents:
             - organizationalUnit
             - top
 
+    ou=aliases:
```

Comment:
From what I remember when managing this, the ldap_scheme.yml is only used during initial Yunohost installation. not regen-conf.

To create the OU on existing installation, I implemented _ensure_ldap_ou_is_created in the alias.py and this does the trick. So it is actually created on first use, at the cost of an additional small ldap request.

Bram already commented that this should ultimately be managed in a migration management framework. I concur.